apis/apis.json

```diff
@@ -446,13 +428,13 @@
     "tags": [ "NLog", "Logging", "Stackdriver" ],
     "dependencies": {
       "NLog": "4.5.0",
-      "Google.Api.Gax.Grpc": "2.5.0",
+      "Google.Api.Gax.Grpc": "default",
```

Comment:
I'm unclear on why this NLog package specifies The "Google.Api.Gax.Grpc" (as default), but the "Google.Cloud.Monitoring.V3" (for example) now doesn't specify a "Google.Api.Gax.Grpc" dependency at all?

They are both beta packages, so I'd expect them to do the same thing?

Reply:
The NLog package is of type "other", so it doesn't get any packages by default. Any "type=grpc" packages get GAX gRPC automatically.

Reply:
OK, yes, that makes sense :)